source/_integrations/my.markdown

```diff
@@ -0,0 +1,24 @@
+---
+title: My Home Assistant
+description: Support for the my.home-assistant.io redirect service.
+ha_category:
+  - Other
+ha_release: 2021.3.0
```

Comment:
This is incorrect, should have been `2021.3`

Reply:
https://github.com/home-assistant/home-assistant.io/pull/16458